### DISCUSSION TITLE
Occupied space

### DISCUSSION THREAD

**Discussion by r0x07k** [milvus-io/milvus#36164] [Q&A and General discussion] [answered]
How do I get information about the overall disk space occupied by a particular collection (data + index)?

**Comment by xiaofan-luan** [comment, marked as answer]
there should be monitoring metrics for that.
Or you can directly check from minio.
What we do is actaully check from S3 inventory on our zilliz cloud

**Comment by r0x07k** [comment]
I see, thank you for explaining that.
